packages/bus/README.md

```diff
@@ -247,6 +231,26 @@ The `HttpOptions<T>` implementation exposes the properties described below:
 | `params`        | Query parameters                                                                           |
 | `createdAt`     | The exact date and time the command was created.                                           |
```

Comment:
Please sort params accordingly: own props first, then inherited props. 

Moreover, descriptions should be updated:

* `url` - Set an absolute URL or path that will be used for the request.
* `method` - Set an HTTP method that is going to be used when making the request.
* `params` - Use to set query parameters.